buildtap.py

```diff
@@ -311,9 +311,7 @@ def package(self):
 
         for f in (os.path.join(self.top, 'COPYING'), os.path.join(self.top, 'COPYRIGHT.GPL')):
             src=open(f, mode='rb')
-            input = src.read()
-            output = input.replace('\n', '\r\n')
-            dst.write(output+' \n')
+            dst.write(src.read()+'\r\n')
```

Comment:
I opened PR https://github.com/OpenVPN/tap-windows6/pull/27 that only has the linefeed fix.